browser/src/index.tsx

```diff
@@ -34,7 +34,7 @@ const start = (args: string[]) => {
 
     // Helper for debugging:
     window["UI"] = UI // tslint:disable-line no-string-literal
-    remote.BrowserWindow.getFocusedWindow().setFullScreen(Config.getValue<boolean>("editor.fullScreenOnStart"))
+    remote.getCurrentWindow().setFullScreen(Config.getValue<boolean>("editor.fullScreenOnStart"))
```

Comment:
FYI - @bert88sta  - just wanted to let you know I had to make this minor change. What happened was that, in some cases, like using `npm run start` if the debugger opened up, and the browser window wasn't the active window, this would crash because `getFocusedWindow` would return null. Minor detail though, just wanted to give you a heads up